openkfw/TruBudget
# 1853 Trim special chars in project title

### Checklist




### Description
- Ampersand ("&") is now translated correctly in project/subproject title and details

### How to test

1. login
2. create project/subproject and add "&" in its title
3. see that "&" is displayed correctly not as &amp ;

Closes #1853
